Automattic/wp-calypso
# SSR: Remove shared directory

Fixes #1551

- Move remaining components back
- Modify NODE_PATHs and webpack configs
- Modify test Makefiles
- Move shared/README to doc/server-side-rendering
- Remove `shared`

To test:
- Restart `make run`
- Check it builds, run e2e tests, click around
